contracts/stellar-axelar-operators/src/interface.rs

```diff
@@ -54,11 +51,10 @@ pub trait AxelarOperatorsInterface {
     /// - `Ok(Val)`: Returns the result of the function execution.
     ///
     /// # Errors
-    /// - `ContractError::NotAnOperator`: If the specified operator is not authorized.
-    /// - Any error propagated from `env.invoke_contract`.
+    /// - [`ContractError::NotAnOperator`]: If the specified operator is not authorized.
     ///
     /// # Authorization
-    /// - The `operator` must authenticate.
+    /// - The `operator` must authorize.
```

Comment:
```suggestion
    /// - An `operator` must authorize.
```
There are multiple operators